### DISCUSSION TITLE
关于架构

### DISCUSSION THREAD

**Discussion by hexWars** [fp-space/HappyDayReminder#1] [Ideas]
我现在有两种设想：
- 第一种是仿造其他的轮子，直接fork或者template生成之后，修改配置就可以运行
- 第二种是，写一个rust的库，上传到crates.io中，然后再开一个仓库，用于birthdayBot的示例，作为template

> 我认为仓库作为template比较好，毕竟有隐私，fork的仓库是没办法私有的

或者是先完成第一种，然后有兴趣再来完成第二种，你觉得呢

@LordFoxFairy

**Comment by hexWars** [comment]
参考项目：
-  https://github.com/kengerlwl/birthdayBot
- https://github.com/Extrader-home/BirthdayReminder

**Reply by LordFoxFairy** [reply]
直接第二种吧，这个功能实现比较简单。
